provisioning/containers/gui-common/novnc/Dockerfile

```diff
@@ -56,4 +52,7 @@ EXPOSE ${HTTP_PORT}
 USER ${USER}
 
 # Start nginx (not in daemon mode for Docker use)
-CMD nginx -g "daemon off;"
+CMD if [ $HIDE_NOVNC_BAR = true ] \
+        then echo "#noVNC_control_bar_anchor {display:none !important;}" \
+        >> $HTML_DATA/app/styles/base.css; fi\
+    nginx -g "daemon off;"
```

Comment:
Here, I would personally move these instructions to a script file and execute that as entrypoint (personal opinion, let me know what do you think).

Second comment, you should probably quote "$HIDE_NOVNC_BAR" to avoid issues if the variable is not set.